### ISSUE TITLE
Error when calling check Balance

### ISSUE THREAD

**Issue by tienvv1234** [ProjectOpenSea/opensea-js#1093]
how can I check the Balance
here is the code I've written following the SDK document 
```
 a = (await openSea.api.getNFTsByCollection('mumbai-collection-10')).nfts;
        console.log('a', a);

        const asset = {
            tokenAddress: a[0].contract, // CryptoKitties
            tokenId: a[0].identifier,
            tokenStandard: 'ERC1155',
        };
        console.log('asset', asset);

        const balance = await openSea.getBalance({
            accountAddress: '0x6E82A097E79251468E54dbE7FE6c943367d96EF0', // string
            asset, // Asset
        });
 ```
 but I it will return to me a error 
 ```
 TypeError: Cannot read properties of undefined (reading 'balanceOf')
    at OpenSeaSDK.<anonymous> (/home/tien/Desktop/javis's works/ntf market place/node_modules/opensea-js/lib/sdk.js:497:22)
    at Generator.next (<anonymous>)
    at /home/tien/Desktop/javis's works/ntf market place/node_modules/opensea-js/lib/sdk.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/tien/Desktop/javis's works/ntf market place/node_modules/opensea-js/lib/sdk.js:4:12)
    at OpenSeaSDK.getBalance (/home/tien/Desktop/javis's works/ntf market place/node_modules/opensea-js/lib/sdk.js:490:16)
    at getAssetFromOpenSea (/home/tien/Desktop/javis's works/ntf market place/index.js:51:39)
 ```

**Comment by ryanio** [comment]
we are looking into fixing this, thanks for reporting

**Comment by ryanio** [comment]
@tienvv1234 we have released v6.1.4 which should have this fixed if you want to give it a try!

**Comment by tienvv1234** [comment]
Yes, it's working, thank you
